### PR TITLE
Remove private residue from public documentation

### PR DESCRIPTION
**Pull request by hollomancer**
## Summary

- replace synthetic team aliases with accountable maintainer metadata
- remove transient AI branch names and personal absolute paths from public artifacts
- relabel uncommitted commercialization results as historical and non-citable
- make the Phase III T6 scorer resolve inputs from the repository's existing data-root environment contract

## Why

Public documentation should describe artifacts an outside reviewer can inspect and reproduce. Branch names, workstation paths, and uncommitted local analyses make canonical docs look like an internal scratchpad and overstate the evidence available in the repository.

## Impact

No analytical behavior changes. The T6 script's defaults are now portable; callers can override the data root with `SBIR_ETL__PATHS__DATA_ROOT`.

## Verification

- `make docs-check`
- `pytest tests/unit/phase3_groundtruth/test_score_t6.py -q` — 5 passed
- Ruff check and format check for `score_t6.py`
- `git diff --check`

### CODE REVIEW

File: docs/commercialization-benchmark-methodology.md [hunk 1]
```diff
@@ -6,11 +6,17 @@ It also documents where our methodology diverges from the authoritative SBA proc
...
 
 The shippable counterpart on `main` is `scripts/run_benchmark.py` + `sbir_etl/models/benchmark_models.py`, which implements the same statutory framework (§638(qq) tier thresholds, 10-FY window, cohort selection) via a different CLI shape (`evaluate` / `sensitivity` / `company` subcommands). It does not currently produce per-firm audit JSON files or do the USAspending + Form D proxy substitution the harness does.
 
-This doc is committed as the **methodology record** so the audit-harness logic is auditable and reproducible, even before the harness itself ships. **Future work**: migrate the harness into `scripts/run_benchmark.py` (or alongside it under `packages/sbir-analytics/`) so the cited file paths resolve and the run is reproducible from `main`. See research-questions.md "Output products" section for the broader status.
+This document is retained as a **historical methodology record**. The described
+audit-harness logic is not auditable from the repository until it is migrated
+into `scripts/run_benchmark.py` (or alongside it under
+`packages/sbir-analytics/`) with committed fixtures and verification. See
+research-questions.md "Output products" for the broader status.
```
Comment (hollomancer) [inline docs/commercialization-benchmark-methodology.md]: **The harness is committed. Both scripts are tracked on this branch, and this same file hands the reader commands to run them.**

```
scripts/archive/data/run_commercialization_benchmark.py   TRACKED, 9009 bytes
scripts/data/audit_one_firm.py                            TRACKED, 8661 bytes
```

So "The audit harness and FY2026 cohort ... are not committed" (line 9-10) is half wrong, and "The described audit-harness logic is not auditable from the repository" (line 16-17) is wrong outright — 17.7 KB of statutory logic is sitting right there.

The file contradicts itself inside its own body. Line 3 names both scripts as the harness; lines 176, 182, and 188 give runnable invocations:

```
.venv/bin/python scripts/archive/data/run_commercialization_benchmark.py --eval-fy 2026
.venv/bin/python scripts/data/audit_one_firm.py <UEI> --eval-fy 2026
```

And `scripts/archive/README.md:33-38` records these as deliberately-kept live tools, not stale archive residue.

What is *genuinely* absent is `reports/validation/commercialization_benchmark_eval_fy2026.csv` — the cohort and the numeric outputs. That's a real and worth-stating limitation: the results can't be reproduced without the cohort. But that's "the inputs and outputs aren't committed," not "the harness isn't committed," and the difference is the whole point of the sentence.

This lands badly precisely because of what the PR is for. An outside reviewer — the exact audience named in the "Why" — reads "not auditable from the repository," skips the committed §638(qq) implementation that is the single most auditable artifact here, and comes away believing there's less to inspect than there is. The PR's goal is inverted by its own banner.

To be fair: the *old* text was wrong the same way. This PR rewrote the sentence and carried the error through rather than introducing it. But rewriting it was the moment to check it.

Suggested replacement for lines 9-11:

> The audit harness is committed (`scripts/archive/data/run_commercialization_benchmark.py`, `scripts/data/audit_one_firm.py`), but the FY2026 cohort and the evaluation output it produced are not. The numerical results below therefore cannot be independently reproduced from this repository and must not be treated as validated or citable outputs.

and drop "is not auditable from the repository" at line 16-17 in favour of naming what's missing.

**Separately, the body still states the results in the present tense**, which the PR body says were relabeled "as historical":
- `:124` — "In our FY2026 run, 2 firms hit this dual-penalty state under net broad: **NanoSonic** ... and **Nanohmics** ..."
- `:149` — "Final cohort match rate: **6 of 143 firms (4%)** ... **zero Tier 1/2 firms have Form D credit**."
- `:155` — "None of our 115 Standard-tier passes need the patent path" — and this line still says "USPTO data on this branch is fixture-only," which is itself transient-branch language of the kind this PR removes elsewhere.

The banner does satisfy the exploratory-tier contract in substance (labeled non-citable), so this isn't a tier violation. But "relabeled as historical" implies the prose moved to past tense, and it didn't.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Review by hollomancer — COMMENTED** [review]
## The scrub is about half done, and one new sentence is factually wrong

The code change is good and every verification claim in the body reproduces exactly. But for a PR whose thesis is "public docs should describe what an outsider can actually inspect," the two sweeps it's named for both stop partway, and the marquee prose change asserts something untrue.

### The one I'd fix before merging

**The new banner says the audit harness "is not committed." Both scripts are tracked** — 9,009 and 8,661 bytes — and the same file gives runnable commands for them. Detail inline. The genuinely-missing artifact is the FY2026 cohort/output CSV, which is a real limitation and worth saying; the harness isn't.

### The scrub misses

**Synthetic aliases — 19 files fixed, 18 left, 6 in the identical `X@project` frontmatter form.** The pass caught `docs@`, `engineering@`, `devops@`, `research@` but not `data@project` or `ml@project`, and never touched the inline `**Owner**: Engineering Team` form:

| Form | Files |
|---|---|
| `Owner: data@project` | `docs/schemas/neo4j.md:4`, `docs/decisions/ADR-003-api-before-mcp.md:4`, `docs/decisions/ADR-004-retire-private-analytics-api.md:3`, `docs/decisions/dspy-evaluation.md:3` |
| `Owner: ml@project` | `docs/ml/cet-integration.md:3`, `docs/transition/cet-integration.md:3` |
| `Owner: data-team` | `docs/data/uspto-data-refresh.md:3` |
| `**Owner**: Engineering Team` | `docs/configuration.md:5`, `docs/development/logging-standards.md:4`, `docs/development/exception-handling.md:4`, `docs/enrichment/sam-gov-integration.md:5`, and five under `docs/testing/` |

**Both templates still teach the abandoned key.** `docs/_template.md:3` and `docs/decisions/_template.md:4` still say `Owner: <owner-email>`, so the next doc created from either reintroduces the split this PR just spent 19 files undoing. No tooling reads `Owner:` or `Maintainer:` — I grepped `tests/`, `scripts/`, `Makefile`, `.github/` — so nothing will ever catch the drift. Updating the two templates is the highest-leverage line in this whole PR.

**The branch-name scrub was `claude/`-only.** Every name the diff removes is a `claude/*` name. Still present:

- `docs/archive/design/2026-07-29-awardee-first-procurement-packet-design.md:215` — `codex/pr450-address-comments`
- same file `:216` — ``all of `agent/monthly-procurement-transition-report` (PR #450)`` — a literal `agent/` name, the exact category the title claims to remove
- `specs/phase3-notice-corpus-fusion/requirements.md:56` — `codex/phase3-methodology-hardening`
- `specs/ma-discovery-integration/design.md:5,60` — `chore/ma-discovery-toolkit`, described as "local branch"

These read as misses rather than deliberate exclusions, since the PR *did* rewrite `specs/dark-majority-resolution/tasks.md:79,95` to strip `claude/nanotech-*` — so archive and spec files were in scope.

**One personal path survives, in a directory the PR otherwise fixed.** `specs/phase3-match-benchmark/mse-dark-phase3.md:89` still has `~/Documents/phase3_adjudication_*`, while `inputs/vehicle_mse_result.json:3-4` in the same directory was de-absolutized. Credit where due: `/Users/`, `C:\Users`, and `/home/<name>/` are otherwise **completely clean** on the branch — main has 4 hits, the branch has 0, so that part of the sweep is genuinely finished.

### The code change is correct

`SBIR_ETL__PATHS__DATA_ROOT` is real, not invented: consumed by `_apply_env_overrides` (`loader.py:93-120`, prefix `SBIR_ETL`, `__` nesting → `config["paths"]["data_root"]`), backed by `config/base.yaml:18` and `schemas/data.py:333`, and already the declared `DATA_ROOT_ENV` in four other modules. All three new defaults match repo convention.

One nit: `score_t6.py:56` defaults to `_WORKTREE_ROOT / "data"` (repo-root absolute) while the canonical `sbir_etl/utils/cloud_storage.py:19-25` and its three redeclarations all default to `"data"` (CWD-relative). So an operator who sets `SBIR_ETL__PATHS__DATA_ROOT=data` to match `config/base.yaml` silently flips this script from repo-anchored to CWD-anchored. I verified the three cases: unset → `/repo/data/raw/...`; `"data"` → `data/raw/...`; `""` → `raw/sbir/...`. Given the module declares `EPISTEMIC_TIER = "exploratory"`, importing `get_data_root` would be over-building — but matching the default costs one line.

Also worth noting the cited test run doesn't cover the change: none of the 5 tests in `test_score_t6.py` touch `DATA_ROOT`, and since it's evaluated at import time, `monkeypatch.setenv` couldn't test it without a reload. The module-level import does prove it loads cleanly, which is the right amount of coverage for exploratory tier — just not verification of this change.

### Clean

Merges cleanly into current main (`git merge-tree` exits 0); none of the 25 files touched by #571/#570/#567/#566 overlap the 30 here. No link breakage — I checked the renamed heading at `research-questions.md:1072` against `_github_heading_slug` in `check_removed_src_references.py:229-241` and the new fragment matches exactly, with no surviving reference to the old anchor. `make docs-check`, the four boundary guards, study-manifest validation, 5 tests, ruff, and `git diff --check` all reproduce as claimed. Adding `Conrad Hollomon` is not new exposure — the name and email are already in `LICENSE`, `README.md`, and all four `pyproject.toml` files on main.

The district-list removal at `research-questions.md:1037-1039` orphans nothing, but it's outside the four bullets in the PR body — worth a mention there.

---
_Generated by [Claude Code](https://claude.ai/code)_
